src/components/home/VolunteerBrief.jsx

```diff
@@ -90,13 +90,14 @@ const VolunteerBrief = () => {
         </Grid>
         <Button
           variant="contained"
-          href="mailto:hugh@codeforpdx.org"
-          target="_blank"
-          rel="noopener"
           sx={{
             mt: '2rem'
           }}
+          href="/volunteer"
         >
+          {/* TODO: This is to determine whether we prefer this button to redirect to Volunteer page 
```

Comment:
Hmmm I didn't get this behavior at all. Tested every way I could think of and went to top of page.

Reply:
That's because for the moment I used `href` rather than the `Link` component from `react-router-dom`, like the Navbar does. I used `href` mostly to demonstrate how it would be to take away the contact modal and see if we like it better.

We can keep it this way or keep it more consistent and change it to `Link`, which would take a bit more finagling.

Reply:
Interesting, I've never encountered that before...

So yeah wrapping this button in a `<Link to='/volunteer'` does indeed keep your scroll on the y-axis when loading that new page/component.

I found these things: https://stackoverflow.com/questions/58431946/why-does-my-react-router-link-bring-me-to-the-middle-of-a-page#:~:text=react%2Drouter%20renders%20a%20new,page%20when%20the%20page%20changes.

https://v5.reactrouter.com/web/guides/scroll-restoration

I'm okay with leaving it as an `<a href=` too.

Reply:
I think ideally our method of routing would be applied consistently across the board but I'm also fine with leaving this as is.

Also, I want to first make sure everyone is in agreement that this button should indeed go to the Volunteer page rather than open the contact modal. I think that's the more intuitive approach but don't want to just make that decision without everyone on board.